pom.xml

```diff
@@ -187,10 +187,6 @@
                             <mainClass>com.autotune.Autotune</mainClass>
                             <name>Autotune</name>
                         </program>
-                        <program>
-                            <mainClass>com.autotune.database.livenessProbe.TestDBConnection</mainClass>
-                            <name>TestDBConnection</name>
-                        </program>
```

Comment:
Is this no longer needed?

Reply:
right now it is not required , when we merge livenessProbe PR  https://github.com/kruize/autotune/pull/691  we can have this change back , Till then we can remove it for now